setup.py

```diff
@@ -72,6 +72,12 @@ def _determine_user_arguments(options):
     )
     if "--with-openmp" in sys.argv:
         sys.argv.remove("--with-openmp")
+    options['intidx_64'] = (
+        '--with-intidx-64' in sys.argv
+        or _get_environment_bool('CI_QUTIP_WITH_INTIDX_64')
+    )
+    if "--with-intidx-64" in sys.argv:
+        sys.argv.remove("--with-intidx-64")
     return options
```

Comment:
If we're going to extend this a lot, at some point we should probably do it properly with `argparse`, but I don't know how well that's going to interact with `setuptools` getting the arguments.

Reply:
I thought something similar. I did a small tidy up for now.